pace-util/pace/util/zarr_monitor.py

```diff
@@ -102,6 +103,7 @@ def _check_writers(self, state):
                 "that were not present in earlier states"
             )
         missing_names = set(self._writers.keys()).difference(state.keys())
+        missing_names.difference_update(self._bypass_checks)
```

Comment:
Grid data only needs to be written once, so you shouldn't need to keep them in `self._writers`. If you never append them into `self._writers` you don't need the _bypass_checks logic.

Reply:
Removed from `self._writers`